packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -22,9 +22,62 @@ export const computeFileToUpdatePath = (root = process.cwd(), defaultFile = 'sty
  * Compare Token variable by name
  * @param a first token variable
  * @param b second token variable
+ * @deprecated use {@link getTokenSorterByName} instead. Will be removed in v13.
  */
 export const compareVariableByName = (a: DesignTokenVariableStructure, b: DesignTokenVariableStructure): number => a.getKey().localeCompare(b.getKey());
 
+/**
+ * Compare Token variable by name
+ * @param _variableSet Complete set of the parsed Design Token
+ */
+export const getTokenSorterByName: TokenListTransform = (_variableSet) => (tokens) => tokens.sort((a, b) => a.getKey().localeCompare(b.getKey()));
+
+
+/**
+ * Compare Token variable by reference to write theme in reference order
```

Comment:
What do you mean by this? Do you sort by crescent order on the depth of the reference? I am not sure I understand the method from the description

Reply:
updated the description